yt_dlp/extractor/zetland.py

```diff
@@ -0,0 +1,73 @@
+            'description': 'md5:9619d426772c133f5abb26db27f26a01',
+            'timestamp': 1705377592,
+            'series_id': '62d54630-e87b-4ab1-a255-8de58dbe1b14',
+        }
+
+    }]
+
+    def _real_extract(self, url):
+        display_id, uploader_id = self._match_valid_url(url).group('id', 'uploader_id')
+        webpage = self._download_webpage(url, display_id)
+
+        next_js_data = self._search_nextjs_data(webpage, display_id)['props']['pageProps']
+        story_data = traverse_obj(next_js_data, ('initialState', 'consume', 'story', 'story'))
+
+        formats = []
+        for audio_url in traverse_obj(story_data, ('story_content', 'meta', 'audioFiles', ...)):
+            formats.append({
+                'url': audio_url,
+                'vcodec': 'none',
+                'ext': determine_ext(audio_url)
+            })
+
+        return merge_dicts({
+            'id': display_id,
+            'formats': formats,
+            'uploader_id': uploader_id
+        }, traverse_obj(story_data, {
+            'title': ('story_content', 'content', 'title') or 'title',
+            'uploader': ('sharer', 'name'),
+            'uploader_id': ('sharer', 'sharer_id'),
+            'description': ('story_content', 'content', 'sosialDescription'),
+            'series_id': ('story_content', 'meta', 'seriesId'),
+            'release_timestamp': ('published_at', {unified_timestamp}),
+            'modified_timestamp': ('revised_at', {unified_timestamp}),
+        }), traverse_obj(next_js_data, ('metaInfo', {
+            'title': ('meta', 'title') or ('ld', 'headline') or ('og', 'og:title') or ('og', 'twitter:title'),
+            'description': (('meta', 'description') or ('ld', 'description')
+                            or ('og', 'og:description') or ('og', 'twitter:description')),
+            'uploader': ('meta', 'author') or ('ld', 'author', 'name'),
+            'uploader_url': ('ld', 'author', 'url'),
+            'thumbnail': ('ld', 'image') or ('og', 'og:image') or ('og', 'twitter:image'),
+            'modified_timestamp': ('ld', 'dateModified', {unified_timestamp}),
+            'release_timestamp': ('ld', 'datePublished', {unified_timestamp}),
+            'timestamp': ('ld', 'dateCreated', {unified_timestamp}),
+        })), {
+            'title': self._html_search_meta(['title', 'og:title', 'twitter:title'], webpage),
+            'description': self._html_search_meta(['description', 'og:description', 'twitter:description'], webpage),
+            'thumbnail': self._html_search_meta(['og:image', 'twitter:image'], webpage),
+            'uploader': self._html_search_meta(['author'], webpage),
+            'release_timestamp': unified_timestamp(self._html_search_meta(['article:published_time'], webpage)),
+        }, self._search_json_ld(webpage, display_id))
```

Comment:
json ld should be non-fatal since it is metadata-only and a fallback to a fallback

Reply:
Resolved in b9aa2b560470266f015de026f4c8ecbbb9591d52